### PR TITLE
Closes #451

### LINKED ISSUE #451: Add JAGS test
Following procedure in https://github.com/Roche/crmPack/pull/424#issuecomment-1336134412

Check if we can pass commands to JAGS executable via minimal `rjags` interface so that we don't have to use `system()` maybe directly or at least not need to find the JAGS executable ourselves. 
Alternative: Suggest `runjags` and use https://github.com/cran/runjags/blob/master/R/utilities.R#L101

### PR DESCRIPTION
**Pull request by wwojciech**
Closes #451

### CODE REVIEW

**Comment by wwojciech** [comment]
Hi @danielinteractive - do you know how to exclude the whole files (dirs) from the linter? I found this https://rdrr.io/cran/lintr/man/exclude.html, but it is not clear how to use this `exclude` fun exactly. Tx!

**Comment by danielinteractive** [comment]
Not sure @wwojciech - @cicdguy any idea for excluding directories from linting?

File: tests/testthat/_jags/data.R [hunk 1]
```diff
@@ -0,0 +1,14 @@
+"betaZ_mean" <- c(0, 1)
```
Comment (danielinteractive) [inline tests/testthat/_jags/data.R:1]: is this file needed? wondering because in the test script you pass this data via list

File: tests/testthat/test-mcmc.R [hunk 2]
```diff
@@ -1,3 +1,61 @@
+# JAGS ----
+
+test_that("JAGS model compile works as expected for an example model", {
+  # The example model here is DualEndpointRW.
+  # The steps undertaken in this unit test correspond to the following commands
+  # in (command line) JAGS:
+  # jags
+  # # and then in an interactive mode
+  # model in "model.bug"
+  # data in "data.R"
```
Comment (danielinteractive) [inline tests/testthat/test-mcmc.R:10]: if the file is needed please explain in more detail which function call below needs it. also explain why the init.R is needed and where etc.

File: tests/testthat/test-mcmc.R [hunk 3]
```diff
@@ -1,3 +1,61 @@
...
+  # compile, nchains(1)
+  # We use JAGS through the rjags R package, so that we do not have to invoke
+  # a system command 'jags' with base::system().
+
+  modfile <- "./_jags/model.bug"
```
Comment (danielinteractive) [inline tests/testthat/test-mcmc.R:15]: not sure if this will work in general reliably? better use https://testthat.r-lib.org/reference/test_path.html

**Review by danielinteractive — COMMENTED** [review]
thanks @wwojciech , please see inside

File: tests/testthat/_jags/data.R (hunk 1 above)
Comment (wwojciech) [reply tests/testthat/_jags/data.R:1]: Yes, actually, none of these 2 `.R` is needed. I added it because I though someone maye use these for direct JAGS access if needed...

File: tests/testthat/test-mcmc.R (hunk 3 above)
Comment (wwojciech) [reply tests/testthat/test-mcmc.R:15]: Sure, Tx!, I will check this.

File: tests/testthat/_jags/data.R (hunk 1 above)
Comment (danielinteractive) [reply tests/testthat/_jags/data.R:1]: then please remove

**Review by danielinteractive — APPROVED** [review]

**Comment by wwojciech** [comment]
Hi @cicdguy - still we have the problem with `Pkgdown` is not skipped but is is hanging. Can we somehow solve it, so that I do not have to force some dummy changes on the related files? Tx!
